logfire/_internal/config.py

```diff
@@ -144,11 +144,14 @@ class PydanticPlugin:
     """Exclude specific modules from instrumentation."""
 
 
-def configure(
+class DeprecatedKwargs(TypedDict):
+    pass
```

Comment:
By "undocumented" you mean you don't have type hints? and that's why you didn't add them here? 🤔

Reply:
yes, added a comment